### PR TITLE
Created portfolio page for Digital Bone Box project

### PR DESCRIPTION
**Pull request by DeviPriya-Kolla**
Created portfolio page for Digital Bone Box project #90

### CODE REVIEW

File: docs/portfolio.md [hunk 1]
```diff
@@ -43,7 +43,9 @@ We are especially interested in projects supporting researchers, focused on rese
   
 **[Where's Religion Desktop](project_wheres_religion_desktop/about.md)** - Where's Religion Desktop is an enhancement on the mobile version which will have extended features.  
   
-**[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event.  
+**[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event. 
+
+**[Digital Bone Box](project_digital_bone_box/about.md)** - This project is a Web Application aims to convert an existing PowerPoint-based educational tool into an interactive, mobile-friendly web application.
```
Comment (kungfuchicken) [inline docs/portfolio.md:48]: This should be in alphabetical order by project name.

File: docs/project_digital_bone_box/about.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+The Digital Bone Box Web App project aims to convert an existing PowerPoint-based educational tool into an interactive, mobile-friendly web application. This app will provide medical students with a comprehensive, easy-to-navigate resource for studying skeletal anatomy. 
+
+The app will categorize the bones into modules such as the bony pelvis, lower limb, upper limb, thorax, skull, and vertebrae. Each module will allow students to explore different bone structures, view detailed information, and interact with labels that identify various parts of the bones. 
+
+### Information
+
+- **Source Code:** -
+- **Client** Dr. Brian Elliot - Program Director - Clinical Health Sciences
+- **Current Tech Lead:** Devi Priya Kolla
```
Comment (kungfuchicken) [inline docs/project_digital_bone_box/about.md:20]: Make sure you get at least your GitHub profile linked here, and possibly your LinkedIn profile. there are examples for the markdown code to do this on the existing pages.

File: docs/project_digital_bone_box/about.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+TBD
+- **Start Date:** Aug, 2024
+- **Technologies Used:**
+  - [HTMX](https://htmx.org/)
+  - Flask
+- **Type:** Web application
+- **License:** [MIT](https://opensource.org/license/mit)
+
+
+### Project Vision
```
Comment (kungfuchicken) [inline docs/project_digital_bone_box/about.md:31]: Please use the standard sections already in use on other pages:

### User Guide
## Technical Information
### Technical Overview
### Architecture
### Development Priorities

We want to be consistent about these sections for OSS branding reasons. You can skip sections you don't have yet, and reformat most of this as items under Development Priorities.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
almost there

File: docs/portfolio.md (hunk 1 above)
Comment (DeviPriya-Kolla) [reply docs/portfolio.md]: Updated the project name according to alphabetical order.

File: docs/project_digital_bone_box/about.md (hunk 2 above)
Comment (DeviPriya-Kolla) [reply docs/project_digital_bone_box/about.md]: Linked my GitHub and LinkedIn profiles.

File: docs/project_digital_bone_box/about.md (hunk 3 above)
Comment (DeviPriya-Kolla) [reply docs/project_digital_bone_box/about.md]: Reformatted the content by following standard sections for OSS branding reasons.

**Review by kungfuchicken — DISMISSED** [review]
lgtm;

**Pull request merged**